src/main/java/com/example/moviemanager/controller/MovieExceptionHandler.java

```diff
@@ -0,0 +1,25 @@
+package com.example.moviemanager.controller;
+
+import com.example.moviemanager.service.MovieNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.time.LocalDateTime;
+
+@ControllerAdvice
+public class MovieExceptionHandler {
```

Comment:
Since this class is related with error handling you can move it to error package

Reply:
Moved

Reply:
Placed 'exception' package inside 'service' package and 'error' package inside controller package.